src/server/game/Warden/WardenWin.cpp

```diff
@@ -39,6 +39,8 @@ static constexpr char _luaEvalPostfix[] = ",'GUILD')end";
 
 static_assert((sizeof(_luaEvalPrefix)-1 + sizeof(_luaEvalMidfix)-1 + sizeof(_luaEvalPostfix)-1 + WARDEN_MAX_LUA_CHECK_LENGTH) == 255);
 
+uint16 _wardenPayloadOffset = 5000;
```

Comment:
Global var.

Reply:
I moved this to the WardenWin class as a static const, is there more appropriate?

Reply:
It's fine there.